lib/util/cache/repository/impl/memory.ts

```diff
@@ -0,0 +1,20 @@
+import type { RepoCache, RepoCacheData } from '../types';
+
+export class MemoryRepoCache implements RepoCache {
+  private data: RepoCacheData = {};
```

Comment:
should we use a protected field on a base class?

Reply:
Okay, done